Master-OB-OpenAPI.json

```
@@ -14446,6 +14420,521 @@
         "items": {
           "$ref": "#/components/schemas/TaskResult"
         }
+      },
+      "ProgramName": {
+        "allOf": [
+          {
+            "$ref": "#/components/schemas/TaxonomyElementString"
+          },
+          {
+            "type": "object",
+            "description": "Name of the Program.",
+            "x-ob-item-type": "StringItemType",
+            "x-ob-item-type-group": "",
+            "x-ob-usage-tips": "Use this field to store the name of the VPP Program that the site is enrolled in",
+            "x-ob-sample-value": {}
+          }
+        ]
+      },
+      "EligibleDeviceType": {
+        "allOf": [
+          {
+            "$ref": "#/components/schemas/TaxonomyElementString"
+          },
+          {
+            "type": "object",
+            "description": "EligibleDeviceType",
```

```suggestion
            "description": "EligibleDeviceType",
```

This one is mysterious

---

@vipulgholapQcells was this an initial approach to something and then was deemed unnecessary? Can I remove this element?